unittests/Utilities/DeleteOldSHMRegions.cpp

```diff
@@ -0,0 +1,43 @@
+#include <fstream>
+#include <stdio.h>
+#include <sys/shm.h>
+#include <unistd.h>
+
+static bool ctime_is_old(uint64_t ctime) {
+  time_t curtime;
+  time(&curtime);
+
+  // If it is older than ten minutes.
+  return (curtime - ctime) > 600;
+}
+
+int main() {
+  // key      shmid perms                  size  cpid  lpid nattch   uid   gid  cuid  cgid      atime      dtime      ctime                   rss                  swap
+  //	 0     360448   777                  4096 165187 165187      0  1002  1002  1002  1002 1679659857 1679659857 1679659857                  4096                     0
+  //	 0      32769   777                  4096 153814 153814      0  1002  1002  1002  1002 1676490841 1676490841 1676490841                     0                  4096
+  std::fstream fs;
```

Comment:
I think this can be shortened to:

```suggestion
  std::ifstream fs{"/proc/sysvipc/shm", std::fstream::binary};
```

Reply:
Resolved.